src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -81,6 +81,7 @@
     final FileDeps fileDeps;
     final InternalProjectConfiguration projectConfiguration;
     final ProcessPaths paths;
+    private final boolean crossCompile;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'crossCompile' could be replaced by a local variable.](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4747622)